docs/integrations/immer.mdx

```diff
@@ -24,17 +24,20 @@ The signature of writeFunction is `(get, set, update: (draft: Draft<Value>) => v
 import { useAtom } from 'jotai'
 import { atomWithImmer } from 'jotai-immer'
 
-const countAtom = atomWithImmer(0)
+const countAtom = atomWithImmer({ value: 0 })
 
 const Counter = () => {
   const [count] = useAtom(countAtom)
-  return <div>count: {count}</div>
+  return <div>count: {count.value}</div>
 }
 
 const Controls = () => {
   const [, setCount] = useAtom(countAtom)
   // setCount === update : (draft: Draft<Value>) => void
-  const inc = () => setCount((c) => (c = c + 1))
```

Comment:
How about `(draft) => ++draft`?

Reply:
This doesn't work for me: https://codesandbox.io/s/crazy-goldstine-jl5vdm

Reply:
Oh, you are absolutely right! How come this has been left for such a long time.